main/libcap-ng/APKBUILD

```diff
@@ -1,11 +1,11 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libcap-ng
-pkgver=0.7.8
-pkgrel=1
-pkgdesc="posix capabilities library"
+pkgver=0.7.9
+pkgrel=0
+pkgdesc="POSIX capabilities library"
 url="http://people.redhat.com/sgrubb/libcap-ng/index.html"
 arch="all"
-license="GPL"
+license="GPL-2.0-or-later GPL-2.1-or-later"
```

Comment:
> GPL-2.1-or-later

There is no such license. You probably mean `LGPL-2.1-or-later`, don't you?

Reply:
Yes, typo fixed.